src/AppBundle/Repository/PerformanceEventRepository.php

```diff
@@ -23,4 +23,13 @@ public function findByDateRangeAndSlug(\DateTime $fromDate, \DateTime $toDate, $
 
         return $query->execute();
     }
+
+    public function FindAllByDate(\DateTime $date)
+    {
+        $query = $this->createQueryBuilder('u')->where('u.dateTime LIKE :date')
```

Comment:
This would be ok
```php
$qb->where('u.dateTime > :startDay')
   ->andWhere('u.dateTime < :endDay')
   ->setParameter('startDay', $date->format('Y-m-d'))
   ->setParameter('endDay', $date->format('Y-m-d'));
```